Doesn't build anymore with the latest nixos-unstable

```
bootlogd> unpacking sources
bootlogd> unpacking source archive /nix/store/a23vzf7b2yrb9hh67kqrdi150lbizmd1-source
bootlogd> source root is source/src
bootlogd> patching sources
bootlogd> updateAutotoolsGnuConfigScriptsPhase
bootlogd> configuring
bootlogd> no configure script, doing nothing
bootlogd> building
bootlogd> build flags: SHELL=/nix/store/a7f7xfp9wyghf44yv6l6fv9dfw492hd3-bash-5.2-p15/bin/bash PREFIX=/nix/store/xaszx0j9a0bhz48qkdp8i99qqv3im65m-bootlogd-2020-02-02
bootlogd> gcc -O2 -Werror -ansi -fomit-frame-pointer -fstack-protector-strong -W -Wall -Wunreachable-code -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -D_XOPEN_SOURCE -D_GNU_SOURCE -DVERSION=\"\"    -c -o bootlogd.o bootlogd.c
bootlogd> <command-line>: error: "_FORTIFY_SOURCE" redefined [-Werror]
bootlogd> <command-line>: note: this is the location of the previous definition
bootlogd> cc1: all warnings being treated as errors
bootlogd> make: *** [<builtin>: bootlogd.o] Error 1
```

--- a/src/Makefile
+++ b/src/Makefile
@@ -14,7 +14,7 @@ VERSION = "2.95.2"
 
 CPPFLAGS =
 CFLAGS  ?= -O2 -Werror
-override CFLAGS += -ansi -fomit-frame-pointer -fstack-protector-strong -W -Wall -Wunreachable-code -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -D_XOPEN_SOURCE -D_GNU_SOURCE -DVERSION=\"$(VERSION)\"
+override CFLAGS += -ansi -fomit-frame-pointer -fstack-protector-strong -W -Wall -Wunreachable-code -Wformat -Werror=format-security -D_XOPEN_SOURCE -D_GNU_SOURCE -DVERSION=\"$(VERSION)\"
 override CFLAGS += $(shell getconf LFS_CFLAGS)
 STATIC	=
 MANDB	:= s@^\('\\\\\"\)[^\*-]*-\*- coding: [^[:blank:]]\+ -\*-@\1@

